api-docs/cloud-load-balancers-v1/api-reference/methods/get-show-current-usage-v1.0-account-loadbalancers-loadbalancerid-usage-current.rst

```diff
@@ -112,7 +88,7 @@ Response
 
     {
         "links": [],
-        "loadBalancerUsageRecords": [
+        "loadBalancerUsageRecord": [
```

Comment:
Both JSON and XML responses seem to be incorrect here. 
Example JSON response I'm seeing. 
`{
    "loadBalancerUsageRecords": [
        {
            "averageNumConnections": 0,
            "averageNumConnectionsSsl": 0,
            "numPolls": 12,
            "sslMode": "ON",
            "outgoingTransfer": 0,
            "incomingTransfer": 0,
            "outgoingTransferSsl": 0,
            "incomingTransferSsl": 0,
            "numVips": 1,
            "endTime": "2019-12-04T21:00:00Z",
            "vipType": "PUBLIC",
            "startTime": "2019-12-04T20:00:00Z",
            "id": 10999135
        },....`